static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -410,6 +411,20 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
     }
   }
 
+  /**
+   * Register listeners so that any active pins are deselected when a user clicks
+   * or focuses on the search bar
+   */
+  setupSearchBarListeners () {
+    const searchBarInput = document.querySelector('#yxt-SearchBar-input--SearchBar');
+    searchBarInput && searchBarInput.addEventListener('click', () => {
```

Comment:
Does this account for clicking on the searchbar button to search?

Reply:
It doesn't, however new searches deselect all results, so the behavior will still occur

Reply:
updated to account for clicking on the search bar button